readme-vars.yml

```diff
@@ -48,6 +48,12 @@ opt_param_env_vars:
   - { env_var: "DB_PORT", env_value: "5432", desc: "PostgreSQL Port" }
   - { env_var: "REDIS_PORT", env_value: "6379", desc: "Redis Port" }  
   - { env_var: "REDIS_PASSWORD", env_value: "", desc: "Redis password" }
+  - { env_var: "REDIS_DBINDEX", env_value: "0", desc: "Redis Database index, integer between 0-15" }
+  - { env_var: "REDIS_SOCKET", env_value: "/var/run/redis/redis.sock", desc: "Redis socket file location" }
+  - { env_var: "DISABLE_REVERSE_GEOCODING", env_value: "false", desc: "Disable reverse Geocoding" }
+  - { env_var: "REVERSE_GEOCODING_PRECISION", env_value: "3", desc: "Reverse geocoding precision (1, 2 & 3)" }
+  - { env_var: "PUBLIC_LOGIN_PAGE_MESSAGE", env_value: "", desc: "Custom message on the login page, should be written in HTML form" }
+  - { env_var: "LOG_LEVEL", env_value: "log", desc: "logging severity levels: simple, verbose, debug, log, warn, error" }
```

Comment:
@martabal sorry to be a nagger, but these vaules are used in the unraid template, `env_value` being the default value, so if someone pulls the unraid template the redis sock will be defined by default and if the variable is not removed then immich will try communicated with the sock rather than tcp

Reply:
Yeah, it would be better to remove it. Just saw they have an [issue related to this env value](https://github.com/immich-app/immich/issues/1364)